Sources/SpeziLLMOpenAI/FunctionCalling/LLMFunctionParameterWrapper.swift

```diff
@@ -6,22 +6,26 @@
 // SPDX-License-Identifier: MIT
 //
 
-import OpenAI
+import OpenAPIRuntime
 
 
+// NOTE: OpenAPIRuntime.OpenAPIObjectContainer is the underlying type for Components.Schemas.FunctionParameters.additionalProperties
+
 /// Alias of the OpenAI `JSONSchema/Property` type, describing properties within an object schema.
-public typealias LLMFunctionParameterPropertySchema = ChatQuery.ChatCompletionToolParam.FunctionDefinition.FunctionParameters.Property
+public typealias LLMFunctionParameterPropertySchema = OpenAPIRuntime.OpenAPIObjectContainer
 /// Alias of the OpenAI `JSONSchema/Item` type, describing array items within an array schema.
-public typealias LLMFunctionParameterItemSchema = ChatQuery.ChatCompletionToolParam.FunctionDefinition.FunctionParameters.Property.Items
+public typealias LLMFunctionParameterItemSchema = OpenAPIRuntime.OpenAPIObjectContainer
 
 
+// swiftlint:disable type_name
 /// Refer to the documentation of ``LLMFunction/Parameter`` for information on how to use the `@Parameter` property wrapper.
 @propertyWrapper
-public class _LLMFunctionParameterWrapper<T: Decodable>: LLMFunctionParameterSchemaCollector { // swiftlint:disable:this type_name
```

Comment:
I would stick with this swiftlint disable syntax, cleaner and more reliable and more maintainable than disabling and enabling it

Reply:
Addressed in: 271b255299326cf23376d82a85f671dea9aa9bf9